code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -529,10 +529,10 @@
 	if(QDELETED(src) || !user || !user.is_holding(src) || user.incapacitated() || !actual_selected_rune)
 		return
 
-	if(HAS_TRAIT(user, TRAIT_FLOORED))
-		to_chat(user, "<span class='cultitalic'>You cannot cast this spell while knocked down!</span>")
-		return
-
+	if(user == target)
+		if(!do_after(user, 3 SECONDS, allow_moving = TRUE, allow_moving_target = TRUE, target = teleportee))
```

Comment:
This allows the user to cancel the spell and prevents people from using multiple of this spell at the same time
```suggestion
		if(!do_after_once(user, 3 SECONDS, allow_moving = TRUE, allow_moving_target = TRUE, target = teleportee))
```

Reply:
Oh, I see, that's what it's for. Thanks!